requirements/requirements.in

```diff
@@ -9,3 +9,4 @@ django-crum
 django-split-settings
 inflection
 sqlparse>=0.5.2 # https://github.com/ansible/django-ansible-base/security/dependabot/9
+dynaconf>=3.2.10,<4.0.0
```

Comment:
It looks like v3 is the latest, is there a known reason why we are pinning and don't want to go to v4? When we pin in this file we usually try to add a comment explaining why so that when someone gets around to trying and update they have some history.

Reply:
The reason is that[ Dynaconf 4.0](https://github.com/dynaconf/dynaconf/milestone/3) will be a breaking change release.

It will introduce type annotated schema, but will deprecate some functionality, I don't expect it to affect what is being used here, but it is better to ensure befire bumping.

The 3.x and 4.x will be maintained together for a while.